cli/src/main/java/com/devonfw/tools/ide/tool/intellij/Intellij.java

```diff
@@ -31,10 +67,31 @@ public boolean install(boolean silent) {
   }
 
   @Override
-  public void installPlugin(PluginDescriptor plugin) {
+  protected void postInstall() {
+
+    super.postInstall();
+    if (this.context.getSystemInfo().isMac()) {
+      setMacOsFilePermissions(getToolPath().resolve("IntelliJ IDEA CE.app").resolve("Contents").resolve("MacOS").resolve(IDEA));
+    }
+  }
 
-    // TODO Auto-generated method stub
+  private static void setMacOsFilePermissions(Path binaryFile) {
 
+    if (Files.exists(binaryFile)) {
+      String permissionStr = FileAccessImpl.generatePermissionString(493);
+      Set<PosixFilePermission> permissions = PosixFilePermissions.fromString(permissionStr);
+      try {
+        Files.setPosixFilePermissions(binaryFile, permissions);
+      } catch (IOException e) {
+        throw new RuntimeException(e);
+      } finally {
+        return;
+      }
+    }
   }
 
-}
\ No newline at end of file
+  @Override
+  public void installPlugin(PluginDescriptor plugin) {
+
```

Comment:
This is not implemented so this PR does not entirely close #25. If you do something like this on purpose, then create a new issue for IntelliJ plugin support and leave a comment here like `TODO https://github.com/devonfw/IDEasy/issues/«id»`. Otherwise we still end up in the same situation that we have currently that some comandlets seem to be present but are incomplete and if we have no issue about the problem, we might forget about it and currently merging this PR would automatically close #25.

Reply:
Adjusted with new issue.